### DISCUSSION TITLE
Gr-satellites and gnuradio 3.8 on windows 10

### DISCUSSION THREAD

**Discussion by Amza09** [daniestevez/gr-satellites#334] [Q&A]
Greetings everyone:)
I would like to ask if there is there is a way to include your "blocks" into a pre installed gnu radio 3.8 on windows 10...

**Comment by daniestevez** [comment]
Hi,
I think it's easier to use Conda to install both GNU Radio and gr-satellites in Windows than trying to add gr-satellites to a GNU Radio installation that has been installed by other means (though it should be possible if you build gr-satellites from source). Instructions about Conda [here](https://wiki.gnuradio.org/index.php?title=CondaInstall) and [here](https://gr-satellites.readthedocs.io/en/latest/installation_conda.html).

**Reply by Amza09** [reply]
> Hi,
> 
> I think it's easier to use Conda to install both GNU Radio and gr-satellites in Windows than trying to add gr-satellites to a GNU Radio installation that has been installed by other means (though it should be possible if you build gr-satellites from source). Instructions about Conda [here](https://wiki.gnuradio.org/index.php?title=CondaInstall) and [here](https://gr-satellites.readthedocs.io/en/latest/installation_conda.html).

Thank you so much for replying 😄 
I actually did that right after I posted the question, unfortunately I am getting the following error:

"DepreciationWarning: distutils version classes are deprecated. Use packaging.version instead."

**Reply by daniestevez** [reply]
I'm not sure if that warning (which seems harmless) comes from gr-satellites or actually from GNU Radio.

**Reply by Amza09** [reply]
> I'm not sure if that warning (which seems harmless) comes from gr-satellites or actually from GNU Radio.

And so it seems; I have noticed similar warnings while running other flow graphs and worked regardless.. So I did clean installation - using Conda - of both gnuradio and gr-satellites, and despite having the same warning, the blocks worked great 😃 thanks again for your replies 🙏🏻
